front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -120,9 +120,15 @@ const StdcmOperationalPoint = ({
     setSearchTerm(e.target.value);
     if (e.target.value.trim().length === 0) {
       dispatchNewPoint(undefined);
+    } else {
+      setSuggestionsVisible(true);
     }
   };
 
+  const handleBlur = () => {
+    setSuggestionsVisible(false);
```

Comment:
I think this should fix the little bug @Akctarus noticed: 

```suggestion
    setSuggestionsVisible(false);
     setSearchTerm('');
```

Reply:
I don't think it's possible. Clicking on a suggestion will select it before instantly deleting the selected value. 
Maybe using something else than onBlur would solve it, but I don't know what else to use.

Reply:
```
  const handleBlur = () => {
    setSuggestionsVisible(false);
    setSearchTerm('');
  };
```

Reply:
Clicking on a suggestion will delete it right after. With the onBlur, selecting a value makes the combo box lose the focus and triggers setSearchTerm(''). 
Maybe we should use another method. 
But the fix still to be working now.